pages/archive/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTeam } from "@/context/team-context";
+import { FolderPlusIcon, PlusIcon } from "lucide-react";
+import ErrorPage from "next/error";
+import { AddDocumentModal } from "@/components/documents/add-document-modal";
+import { AddFolderModal } from "@/components/folders/add-folder-modal";
+import AppLayout from "@/components/layouts/app";
+import { Button } from "@/components/ui/button";
+import { Separator } from "@/components/ui/separator";
+
+import useDocuments, { useRootFolders } from "@/lib/swr/use-documents";
+import { ArchivedDocumentsList } from "@/components/documents/archived-document-list";
+
+export default function Documents() {
+  const { documents,error } = useDocuments();
```

Comment:
i would expect to only fetch archived documents here, not all documents. so you could add a variable to the `useDocuments` swr hook and then only fetch the archived documents.

Reply:
> i would expect to only fetch archived documents here, not all documents. so you could add a variable to the `useDocuments` swr hook and then only fetch the archived documents.

@mfts As yout told, I did like this
![image](https://github.com/user-attachments/assets/0835811c-ff7c-4b48-9484-9909b16274da)
using boolean variable isArchived